.settings.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings>
```

Comment:
Why do we need this file?
If we'd have the reason in the commit message I wouldn't ask such a question.
Thanks